conans/client/cmake.py

```diff
@@ -128,9 +133,15 @@ def flags(self):
                 flags.extend(["-DCONAN_CXX_FLAGS=-m32",
                               "-DCONAN_SHARED_LINKER_FLAGS=-m32",
                               "-DCONAN_C_FLAGS=-m32"])
+            elif op_system == "Windows" and host_platform != op_system and comp == "gcc":
```

Comment:
I think it should be better to group these flags in the first comparison in Line 125, so the compiler is set there, for cross-building to Win. Think like to refactor the flags for cross compiling to Win in one separate function (not necessary, but at least the idea).

And leave this section here for CXX_FLAGS.